index.html

```diff
@@ -416,65 +416,172 @@ <h2 id="liaisons">
       the following understanding: W3C will seek and expect an organizational
       commitment under the CLA starting with the individual's first request to
-      make a contribution to <a href="#deliverables">Deliverables</a>. The
+      make a contribution to a group <a href="#deliverables">Deliverable</a>. The
       section on <a href="#contrib">Contribution Mechanics</a> describes how
       W3C expects to monitor these contribution requests.
     </p>
+
+   <p class="note">Proposed charter deleted this paragraph</p>  
+    <h2 id="worklimit">
+      Work Limited to Charter Scope
+    </h2>
+    
+    <p>The group will not publish Specifications on topics other than those 
+    listed under <a href="#specifications">Specifications</a> above. 
+    See below for <a href="#charter-change">how to modify the charter</a>. 
+    </p>
+    
     <h2 id="contrib">
       Contribution Mechanics
     </h2>
+    
     <p>
-      Community Group participants agree to make contributions in the GitHub
-      repository for the project that they are interested in. This may be in
-      the form of a pull request (preferred), by raising an issue, or by adding
-      a comment to an existing issue.
+    Substantive Contributions to Specifications can only be made by Community 
+    Group Participants who have agreed to the 
+    <a href="http://www.w3.org/community/about/agreements/cla/">W3C Community
+      Contributor License Agreement (CLA)</a>. 
     </p>
+    
     <p>
-      Specifications created for proposals in the Community Group must use the
+      Specifications created in the Community Group must use the
       <a href=
       "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">
-      W3C Software and Document License</a>.
+      W3C Software and Document License</a>.  All other documents produced 
+      by the group should use that License where possible. 
     </p>
+
+    <p class="remove">
+      {TBD: if CG doesn't use GitHub replace the remaining paragraphs in this 
+      section with: "All Contributions are made on the groups public mail list
+      or public contrib list"} 
+    </p>
+    
+    <p>
+      Community Group participants agree to make all contributions 
+      in the GitHub repo the group is using for the particular document. 
+      This may be in the form of a pull request (preferred), by raising an issue, 
+      or by adding a comment to an existing issue.
+    </p>
+
     <p>
       All Github repositories attached to the Community Group must contain a
-      copy of the <a href="CONTRIBUTING.md">CONTRIBUTING</a> and <a href=
-      "LICENSE.md">LICENSE</a> files.
+      copy of the 
+      <a href="https://github.com/w3c/licenses/blob/master/CG-CONTRIBUTING.md">CONTRIBUTING</a>
+      and <a href=
+      "https://github.com/w3c/licenses/blob/master/CG-LICENSE.md">LICENSE</a> files.
+    </p>
+   <p class="note">Proposed charter deleted last section of this paragraph</p>
```

Comment:
We removed it because you had removed one line, so I asked to remove that line to match you.  And we decided the other line in the paragraph isn't needed either.  So we removed both of them.